samples/suit/smp_transfer/README.rst

```diff
@@ -55,27 +55,29 @@ Configuration
 |config|
 
 The default configuration uses UART with sequence number 1 (shown as Version 1 in the nRF Device Manager app).
-To change the sequence number of the application, configure the ``SB_CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM`` sysbuild Kconfig option.
-This also changes the number of blinks on **LED 0** and sets the :ref:`sequence number <ug_suit_dfu_suit_manifest_elements>` of the :ref:`SUIT envelope <ug_suit_dfu_suit_concepts>`’s manifest.
+To change the sequence number of the application, configure the ``APP_ROOT_SEQ_NUM`` inside the the :file:`VERSION` file, used for :ref:`zephyr:app-version-details` in Zephyr and the |NCS|.
+It sets the :ref:`sequence number <ug_suit_dfu_suit_manifest_elements>` of the :ref:`SUIT envelope <ug_suit_dfu_suit_concepts>`’s manifest.
+To change the number of blinks on **LED 0**, configure the ``CONFIG_N_BLINKS`` Kconfig option.
 
-To use this configuration, build the sample with :ref:`configuration_system_overview_sysbuild` and set the ``SB_CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM`` sysbuild Kconfig option to ``x``, where ``x`` is the version number.
+To use this configuration, set the sequence number by appending ``APP_ROOT_SEQ_NUM = <x>`` inside the :file:`VERSION` file, build the sample with :ref:`configuration_system_overview_sysbuild` and set the ``CONFIG_N_BLINKS`` Kconfig option to ``x``, where ``x`` is the sequence number number.
```

Comment:
```suggestion
To use this configuration:

1. Set the sequence number by appending ``APP_ROOT_SEQ_NUM = <x>`` inside the :file:`VERSION` file.
#. Set the ``CONFIG_N_BLINKS`` Kconfig option to ``x``, where ``x`` is the sequence number number.
#. Build the sample with :ref:`configuration_system_overview_sysbuild`.    
```

Reply:
Section reworked to remove the unclear sample.